src/components/organisms/ProfileModal/EditProfileForm/EditProfileForm.tsx

```diff
@@ -99,6 +103,33 @@ export const EditProfileForm: React.FunctionComponent<EditProfileFormProps> = ({
           ref={register()}
         />
 
+        <div className="EditProfileForm__edit-input-title">Real name</div>
+        <InputField
+          containerClassName="EditProfileForm__input-container-name"
+          inputClassName="EditProfileForm__input-name"
+          name="realName"
+          placeholder="Your real name"
+          ref={register()}
+        />
+
+        <div className="EditProfileForm__edit-input-title">Department</div>
+        <InputField
+          containerClassName="EditProfileForm__input-container-name"
+          inputClassName="EditProfileForm__input-name"
+          name="companyDepartment"
+          placeholder="Your department"
+          ref={register()}
+        />
+
+        <div className="EditProfileForm__edit-input-title">Title</div>
+        <InputField
+          containerClassName="EditProfileForm__input-container-name"
+          inputClassName="EditProfileForm__input-name"
+          name="companyTitle"
+          placeholder="Your title"
+          ref={register()}
+        />
```

Comment:
Instead of using `div` for the ‘titles’ of these edit fields, can you use `label`? And if so, can you nest the `InputField` within them? That way it will use proper semantic html/accessibility/etc

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1667/commits/b1a496133c781e522058ff4474875191d420b179